### PR TITLE
fix: add hidden attr to csrf inputs

### PR DESCRIPTION
**Pull request by yordis**
fixes phoenixframework/phoenix#5197

### CODE REVIEW

**Comment by josevalim** [comment]
@yordis can you please check the tests? Thank you!

**Comment by yordis** [comment]
@josevalim yeah I was adding the test, a habit of mind to push as soon as possible so CI caches things ... I need to configure it to make it Draft by default.

File: test/phoenix_live_view/diff_test.exs [hunk 1]
```diff
@@ -831,7 +831,7 @@ defmodule Phoenix.LiveView.DiffTest do
       assert {_, _, 3} = components
 
       assigns = %{socket: %Socket{}, if: false}
-      {socket, full_render, components} = render(conditional_slot_tracking(assigns))
+      {_socket, _full_render, components} = render(conditional_slot_tracking(assigns))
```
Comment (yordis) [inline test/phoenix_live_view/diff_test.exs:834]: fixed some warnings

**Comment by yordis** [comment]
@josevalim ready!

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:

**Pull request merged**